cypress/page-objects/pastPerfectPage/PastPerfectEle.js

```diff
@@ -0,0 +1,22 @@
+class PastPerfectEle {
+
+    open() {
+        return '/verbs/conjugation/past-perfect/';
+    }
+
+    getHeader() {
+        return '#content h1';
+    }
+
+    getSpecial1() {
```

Comment:
if you want we can improve this selector and parse the number of the symbol number like so:

```
    getSpecialSymbolWithNumber(specialSymbolNumber) {
        return `[onclick="add_special${specialSymbolNumber}(\'ß\')"]`;
    }
```